modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -47,46 +47,32 @@ Built-in dashboards include various pages to view different types of charts:
 * JVM - Heap memory utilization
 * JVM - Thread count
 
-//* Response Time by App: Number of milliseconds (ms) required for each response.
-//* Messages Processed: Number of messages at a given time.
-//* Response Time by Inbound Endpoint: Number of milliseconds (ms) for a response from inbound endpoints to your Mule app.
-//* Response Time by Outbound Endpoint: Number of milliseconds (ms) for a response from outbound endpoints of your Mule app.
-//* CPU: Percentage of CPU used over time. The percentages are calculated per worker and as the total for all workers.
-//* Memory: Number of mebibytes (MiB) used over time.
-| *Inbound* a|
+| <<inbound-charts>> a|
 
 * Total Inbound Requests by Response Type
 * Average Response Time
 * Average Response Time by Endpoint
```

Comment:
This is no longer just "Average" Response Time by Endpoint. We have min / max / avg in this table. So maybe just take out the word "Average".

Reply:
```suggestion
* Response Time by Endpoint
```